distributions/Dist_mh3908.py

```diff
@@ -3,22 +3,22 @@
 
 class Dist_mh3908(BaseDistribution):
 	def __init__(self):
-		self.f_max = 25
-		self.x_min = 100
-		self.x_max = 200
+		self.f_max = 8.707
+		self.x_min = 0.5
+		self.x_max = 1.5
 
 
 	def pdf(self, x):
 		"""This is your PDF"""
-		return (x**3)+np.log(x)
+		return ((x**(1./2))+(1/(x**3)))
 
 	def mean(self):
 		"""This is the mean of the PDF"""
-		return 150.
+		return 2.36489
 
 	def std(self):
 		"""This is the standard deviation of the pdf"""
-		return 38.
+		return sqrt(6.54295)
```

Comment:
They come from the integral calculations for the mean and variance. I'm not sure how to program that in python.